### ISSUE TITLE
Error in the lastest version: TypeError: Cannot read property '11111' of undefined.

### ISSUE THREAD

**Issue by wizardnet972** [vuex-orm/vuex-orm#444]
I got an error in runtime on lastest version (package.json below):
```
TypeError: Cannot read property '11111' of undefined
 at BelongsTo.attach (vuex-orm.esm.js?0937:1156)
    at eval (vuex-orm.esm.js?0937:3307)
    at eval (vuex-orm.esm.js?0937:637)
    at Array.forEach (<anonymous>)
    at Object.forOwn (vuex-orm.esm.js?0937:637)
    at eval (vuex-orm.esm.js?0937:3304)
    at eval (vuex-orm.esm.js?0937:637)
    at Array.forEach (<anonymous>)
    at Object.forOwn (vuex-orm.esm.js?0937:637)
    at eval (vuex-orm.esm.js?0937:3303)
```
**Reproduction Link**
https://github.com/wizardnet972/vuex-orm-ts

**Steps to reproduce**
I created article model:
```
import { Model } from '@vuex-orm/core';
import { Category } from './category';

export class Article extends Model {
  public static entity = 'articles';

  public static primaryKey = '_id';

  public static fields() {
    return {
      _id: this.attr(null),
      article_id: this.attr(null),
      name: this.attr(''),
      category: this.belongsTo(Category, 'category_id'),
      category_id: this.attr(''),
      text: this.attr(null),
      metadata: this.attr(null),
    };
  }
}
```
and category model:
```
import { Model } from '@vuex-orm/core';

export interface CategoryModel {
  _id?: string;
  name: string;
}

export class Category extends Model {
  public static entity = 'categories';

  public static primaryKey = '_id';

  public static fields() {
    return {
      _id: this.attr(null),
      name: this.attr(''),
    };
  }
}
```

and in helloworld component I write:
```
public categories = null;

async mounted() {
    const categories = [
      { _id: "11111", name: "cat1" },
      { _id: "22222", name: "cat1" }
    ];

    const articles = [{ _id: "AAAA", name: "article1", category: "11111" }];

    await Category.insertOrUpdate({ data: categories });
    await Article.insertOrUpdate({ data: articles });

    this.categories = Category.query().all();
  }
```

This is was working. but now I get an error.

**package.json:**
```
  "dependencies": {
    "@vuex-orm/core": "^0.32.2",
    "core-js": "^2.6.5",
    "register-service-worker": "^1.6.2",
    "vue": "^2.6.10",
    "vue-class-component": "^7.0.2",
    "vue-property-decorator": "^8.1.0",
    "vue-router": "^3.0.3",
    "vuex": "^3.0.1"
  },
  "devDependencies": {
    "@types/jest": "^23.1.4",
    "@vue/cli-plugin-babel": "^3.11.0",
    "@vue/cli-plugin-e2e-cypress": "^3.11.0",
    "@vue/cli-plugin-pwa": "^3.11.0",
    "@vue/cli-plugin-typescript": "^3.11.0",
    "@vue/cli-plugin-unit-jest": "^3.11.0",
    "@vue/cli-service": "^3.11.0",
    "@vue/test-utils": "1.0.0-beta.29",
    "babel-core": "7.0.0-bridge.0",
    "lint-staged": "^8.1.5",
    "sass": "^1.18.0",
    "sass-loader": "^7.1.0",
    "ts-jest": "^23.0.0",
    "typescript": "^3.4.3",
    "vue-template-compiler": "^2.6.10"
  },
```

**Comment by eldomagan** [comment]
I have the same issue. It happen when you are inserting data with belongsTo or hasOne relation set to related key. A workaround would be insert articles like that:

```js
Article.insertOrUpdate({
  data: [{ _id: "AAAA", name: "article1", category: {  id: "11111" } }]
})
```

**Comment by kiaking** [comment]
The issue has been resolved at #446 thanks to @eldomagan ! I'll be making a new release soon.

**Comment by kiaking** [comment]
Sorry it took long. This issue is now fixed at Vuex ORM 0.32.3! 🔥 
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.32.3
